cmd/agent/app/integrations.go

```diff
@@ -749,6 +749,19 @@ func moveConfigurationFiles(srcFolder string, dstFolder string) error {
 	errorMsg := ""
 	for _, file := range files {
 		filename := file.Name()
+		// Copy SNMP profiles
+		if filename == "profiles" {
+			profileDest := filepath.Join(dstFolder, "profiles")
+			if err = os.MkdirAll(profileDest, os.ModeDir|0755); err != nil {
+				return err
```

Comment:
Should we use the same error handling strategy than here? Saving error string to `errorMsg` and `continue`:
https://github.com/DataDog/datadog-agent/blob/master/cmd/agent/app/integrations.go#L758-L767